templates/registration/contact.html

```diff
@@ -2,9 +2,14 @@
 
 {% block content %}
     <h1>Contact Form</h1>
+    <div class="break">
+        <hr>
+    </div>
+    <div class="contact-form">
     <form action="" method="POST">
```

Comment:
Would you mind indenting the whole form? In Sublime you can do this easily by selecting lines 9-13 and then just pressing the TAB key.